src/Process/Runner/MutationTestingRunner.php

```diff
@@ -132,6 +132,7 @@ public function run(iterable $mutations, string $testFrameworkExtraOptions): voi
                 return true;
             })
             ->filter(function (Mutant $mutant) {
+                // TODO refactor this comparison into a dedicated comparer to make it possible to swap strategies
```

Comment:
For example, if a time limit set to 25 seconds, but a whole test suite takes only 20 seconds, we can tell that any set of tests will take less than the timeout. Therefore we can safely skip this check.

Reply:
> For example, if a time limit set to 25 seconds, but a whole test suite takes only 20 seconds, we can tell that any set of tests will take less than the timeout. Therefore we can safely skip this check.

where we can get this information from, not calling `getNominalTestExecutionTime()`?

Reply:
From the actual time spent on the initial test run, or from the same junit report. There are multiple opportunities.

Reply:
👍  I had some along those lines in mind as well